srcpkgs/vulkan-loader/template

```diff
@@ -13,4 +13,10 @@ maintainer="tibequadorian <tibequadorian@posteo.de>"
 license="Apache-2.0"
 homepage="https://www.khronos.org/vulkan/"
 distfiles="https://github.com/KhronosGroup/Vulkan-Loader/archive/sdk-${version}.tar.gz"
-checksum=5dcd42b564804f4c01891ddd429a4a5028113c4cf39b44256689e8f543773134
+checksum=d9d103a6481a19ab9c9b880c1264e3026931741d3184b9930157551453bd4f92
+
+if [ "$XBPS_CROSS_BUILD" ]; then
+	# https://github.com/KhronosGroup/Vulkan-Loader/issues/881
+	# Fixed in master, should be removed in next release
```

Comment:
Couldn't we include https://github.com/KhronosGroup/Vulkan-Loader/commit/d12ff7d48181236b47f808173e044a11423c0f26 here as a patch in the meanwhile before a new sdk tag? I tested locally and it applied just fine with cross-build for `aarch64-glibc` and `aarch64-musl` working :)

Reply:
Sure, thank you for testing!